management-account/terraform/s3.tf

```diff
@@ -465,3 +465,42 @@ module "focus_reports_s3_bucket" {
     }
   }
 }
+
+# moj-cur-reports-greenops without kms 
+module "cur_v2_hourly_without_kms" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+
+  source  = "terraform-aws-modules/s3-bucket/aws"
+  version = "4.3.0"
```

Comment:
Looking at the rest of this file, is there a reason you've switched to using a community module instead of the internal module?